features/brokers/create_and_manage_quotes.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Create and Manage Quotes
+  In order for Brokers to be able to send a quote to an employe to claim
+  The broker should be able to generate a quote
+
+  Background: Set up site and broker
+    Given a CCA site exists with a benefit market
+    Given there is a Broker XYZ Broking
+    And the person is assigned to a broker agency
```

Comment:
Can we change person to broker so its easier for new developers to follow

Reply:
Will do it